cmd/nerdctl/image_inspect.go

```diff
@@ -42,6 +44,11 @@ var imageInspectCommand = &cli.Command{
 			Usage: "Inspect mode, \"dockercompat\" for Docker-compatible output, \"native\" for containerd-native output",
 			Value: "dockercompat",
 		},
+		&cli.StringFlag{
+			Name:    "format",
+			Aliases: []string{"f"},
+			Usage:   "Format the output using the given Go template, e.g, '{{json .}}'",
```

Comment:
Please update `README.md` too

Reply:
ContainerInspectBashComplete and  imageInspectBashComplete func are the same , is it necessary to put them into fmtutil.go?